conf/agent/agent_full.conf

```diff
@@ -82,6 +82,13 @@ agent {
     
     # allowed_foreign_jwt_claims: set a list of trusted claims to be returned when validating foreign JWTSVIDs
     # allowed_foreign_jwt_claims = []
+
+    # experimental: The experimental options that are subject to change or removal
+    # experimental {
+    #     # named_pipe_name: Pipe name to bind the SPIRE Agent API named pipe (Windows only).
+    #     # Default: \spire-agent\public\api
+    #     named_pipe_name = "\\spire-agent\\public\\api"
```

Comment:
Do the backslashes need to be escaped like this for the conf file?

Reply:
Yes, otherwise you will get an "illegal char escape" error:
https://github.com/hashicorp/hcl/blob/v1.0.1-vault-3/hcl/scanner/scanner.go#L532